src/llm_utils.py

```diff
@@ -54,18 +54,13 @@ def analyze_code_changes(diff_content: str) -> List[Dict]:
         model="gemini-2.0-flash", contents=prompt
     )   
 
-    #print(response.text)
+    print(response.text)
     
     # Parse and format the response
     review_comments = parse_llm_response(response.text)
     return review_comments
 
```

Comment:
This comment is unnecessary.

```suggestion
Remove the comment.
```